backend/api/Controllers/ParcelController.cs

```diff
@@ -129,12 +129,12 @@ public IActionResult AddParcel([FromBody] Model.ParcelModel model)
         /// <param name="id"></param>
         /// <param name="model"></param>
         /// <returns></returns>
-        [HttpPut("{id}")]
+        [HttpPut()]
         [HasPermission(Permissions.PropertyEdit)]
         [Produces("application/json")]
         [ProducesResponseType(typeof(Model.ParcelModel), 200)]
         [SwaggerOperation(Tags = new[] { "parcel" })]
-        public IActionResult UpdateParcel(int id, [FromBody] Model.ParcelModel model)
+        public IActionResult UpdateParcel([FromBody] Model.ParcelModel model)
```

Comment:
I changed this to standardize our routes `/parcels/{id}`.  This was based on our discussion.  I know it isn't technically required in-terms of implementation, but I do agree it would best to keep them standardized.